doc/source/users_guide/cime-change-namelist.rst

```diff
@@ -50,11 +55,140 @@ On the hand, to change the driver namelist value of the starting year/month/day,
 Note that
 
 To see the result of change, call `preview_namelists <../Tools_user/preview_namelists.html>`_  and verify that the new value appears in **CaseDocs/drv_in**.
+
+.. _basic_example:
+
+Setting up a multi-year run
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This shows all of the steps necessary to do a multi-year simulation starting from a "cold start" for all components.  The
+compset and resolution in this example are for CESM but the steps are similar for other compets.
```

Comment:
similar for other models.  not compets

Reply:
If you were going to run a F-case for 11 years, the steps would be the same.